runtime/doc/lua.txt

```diff
@@ -4624,6 +4624,31 @@ vim.text.hexencode({str})                               *vim.text.hexencode()*
     Return: ~
         (`string`) Hex encoded string
 
+vim.text.indent({size}, {text}, {opts})                    *vim.text.indent()*
+    Sets the common leading whitespace of every non-blank line in `text` to
+    `size` spaces.
+
+    Leading TAB chars are replaced by spaces (default: tabwidth=4).
+
+    To "dedent" (remove the common indent), pass `size=0`. >lua
+        local indented, old_indent = vim.text.indent(0, '')
+        indented = vim.text.indent(old_indent + 2, '')
+<
+
+    To adjust an existing indent, call indent() twice: >lua
```

Comment:
`g==` was valuable because it encouraged me to check that these examples work :)